### PR TITLE
Do not show collection icon for pending connections

### PR DESCRIPTION
**Pull request by mikeger**
- Do not show collection buttons for pending incoming or outgoing connections

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+NavigationBar.swift [hunk 1]
```diff
@@ -90,9 +90,13 @@ public extension ConversationViewController {
             backButton.hitAreaPadding = CGSize(width: 28, height: 20)
             items.append(UIBarButtonItem(customView: backButton))
         }
-        let collectionsButton = collectionsBarButtonItem
-        collectionsButton.hitAreaPadding = CGSize(width: 0, height: 20)
-        items.append(UIBarButtonItem(customView: collectionsButton))
+        
+        if let connection = conversation.connection, connection.status != .pending && connection.status != .sent {
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+NavigationBar.swift]: Will this reload when a connection is accepted?

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+NavigationBar.swift]: Good question, it is not reloaded, but also not likely that you would have a lot of content right after accepting the connection. I would check with QA and design.

Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+NavigationBar.swift]: Seems same logic would apply to calling buttons. How's that handled atm?

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+NavigationBar.swift]: I update it now the same way the calling buttons are updated: https://github.com/wireapp/wire-ios/pull/580/files#diff-d0adf7620aab7082451935f800968beaR889

**Pull request merged**